test/e2e/kubevirt.go

```diff
@@ -971,12 +995,29 @@ passwd:
 			return pods
 		}
 
+		iperfServerScript = `
+dnf install -y psmisc procps
+iface=$(ifconfig  |grep flags |grep -v "eth0\|lo" | sed "s/: .*//")
+ipv4=$(ifconfig $iface | grep "inet "|awk '{print $2}'| sed "s#/.*##")
+ipv6=$(ifconfig $iface | grep inet6 |grep -v fe80 |awk '{print $2}'| sed "s#/.*##")
+if [ "$ipv4" != "" ]; then
+	iperf3 -s -D --bind $ipv4 --logfile /tmp/test_${ipv4}_iperf3.log
+fi
+if [ "$ipv6" != "" ]; then
+	iperf3 -s -D --bind $ipv6 --logfile /tmp/test_${ipv6}_iperf3.log
```

Comment:
That complicate stuff since then you have to check if something has die inside the container or create ddifferent containers per family to check container state.

Reply:
I'd prefer to have a container per family over having to rely on the string comparison you're making further on: `Not(ContainSubstring("0.00 Bytes  0.00 bits/sec")),`

Reply:
Even with that we need the string comparation to check stuff like downtime later on.